src/app/components/tree-view/index.html

```diff
@@ -126,20 +126,20 @@ <h3>Components</h3>
       sectionHeading="Angular tree view components"
     >
       <p>
-        To apply the SKY UX themes and additional functionality to the <sky-code>angular-tree-component</sky-code>, you must include the following components.
+        <sky-code>SkyAngularTreeModule</sky-code> provides SKY UX components and styles to complement the Angular tree component and apply SKY UX themes and functionality to hierarchical list views. For information about the Angular tree component and its capabilities, see the <a href="https://angular2-tree.readme.io/docs">Angular tree component documentation</a>.
       </p>
 
       <sky-demo-page-property
         propertyName="sky-angular-tree-wrapper"
```

Comment:
How do users control whether to display the context menu? We have a component in the module, but we don’t document it in the Development tab. I think Alex said the toolbar component is for internal use only, but I don’t remember if that’s also the case for context menu. If not, I think we need to document that component. But if so, how do consumers of this module determine whether to display the context menu? It seems like we need something about this somewhere. If not, should we remove it as an option in the demo and Design docs?

Reply:
Yes, we could use some clarity here. There's no property or setting for a consumer to use context menus. They simply just include the dropdown component inside the `sky-angular-tree-node`. We've added built-in support for the dropdown, such that we'll make sure it gets styled and positioned correctly in the node. If the consumer adds it to the template, we do the rest.

What's the best way to describe that?

Reply:
It seems like we should include this info in the Design docs. We have a section there about Active state that is tied to the split view component, and it seems like this. What do you guys think of adding a section under *Options* along the lines of:

*Context menu*
When users need to perform actions on items in hierarchical lists, you can place dropdown components in the tree node components for those items. The tree node component automatically handles handles styling and positioning for the context menu.

Reply:
Love it. :shipit:

Reply:
Good deal. I'll make this update and double-check the naming tonight. ... @Blackbaud-ToddRoberts, let me know if you have any edits here.

Reply:
@Blackbaud-AdamFunderburk brought up that we should be sure the code example makes it clear how to do this

Reply:
I added a new section with the text in this conversation and linked to the dropdown component docs. ... The demo includes an option to turn context menus off and on, and I see a dropdown component in the code samples. @Blackbaud-AlexKingman, do you think what we have is sufficient to address Adam's point?

Reply:
Yep, I think we're covered here.

Reply:
@Blackbaud-ToddRoberts, are you good with no image?

Reply:
Resolving this thread because Todd took care of the updates to the Design tab, and I added context menu info to the Development tab.